docs/usage/triggers/command.md

```diff
@@ -11,12 +11,16 @@ grand_parent: Usage
 # Received_Command
 
 
-| Options  | Description                                                          | Example                            |
-| -------- | -------------------------------------------------------------------- | ---------------------------------- |
-| command  | Only execute rule if the command matches this/these command/commands | `command: 7` or `commands: [7,14]` |
-| commands | Alias of command, may be used if matching more than one command      | `commands: [7,14]`                 |
+| Options  | Description                                                          | Example                                  |
+| -------- | -------------------------------------------------------------------- | ---------------------------------------- |
+| command  | Only execute rule if the command matches this/these command/commands | `command: 7` or  `command: ->c {c.odd?}` |
+| commands | Alias of command, may be used if matching more than one command      | `commands: [7,14]` or  `to: 7..14`       |
 
-The `command` value restricts the rule from running to only if the command matches
+The `command` or `commands` option restricts the rule from running only if the command matches the supplied arguments
+1. The `command` or `commands` option supports ranges
+2. The `command` or `commands` option supports procs
```

Comment:
Don't forget to mention that it can also accept an array of values. We may be taking it for granted.